tests/parser.cpp

```diff
@@ -1762,7 +1763,8 @@ TEST(Parser, offsetof_expression)
        "    int: 0\n"
        "  offsetof: \n"
        "   dereference\n"
-       "    variable: $foo\n");
+       "    variable: $foo\n"
+       "    field: x\n");
```

Comment:
No need for the additional `++depth_` in the printer - `field` should be indented to the same level as the expr (dereference here) as it's a direct child of the `offsetof`